doc/articles/get-started-vs-2022.md

```diff
@@ -48,7 +48,14 @@ To create Uno Platform applications you will need [**Visual Studio 2022 17.4 or
 ## Create an application
 
 To create an Uno Platform app:
-1. Create a new C# solution using the **Uno Platform App** template, from Visual Studio's **Start Page**
+1. Create a new C# solution using the **Uno Platform App** template, from Visual Studio's **Start Page**, then click the **Next** button
+
+    ![](Assets/tutorial01/create-unoplatform-app.PNG)
```

Comment:
Consider adding an alt text (for this image and the below image as well)

Reply:
Thanks, good advice @Youssef1313 :)
The changes are done and I also did it for the other images of the document